clients/openshift.go

```diff
@@ -265,12 +141,7 @@ func (o OpenshiftClient) GetClusterNetworkPlugin() (string, error) {
 
 // GetNetNamespace - Get Net Namespace.
 func (o OpenshiftClient) GetNetNamespace(nsName string) (*networkoapi.NetNamespace, error) {
-	netNamespace := &networkoapi.NetNamespace{}
-	err := o.networkClient.Get().Resource("netnamespaces").Name(nsName).Do().Into(netNamespace)
-	if err != nil {
-		return nil, err
-	}
-	return netNamespace, nil
+	return o.networkClient.NetNamespaces().Get(nsName, metav1.GetOptions{})
```

Comment:
Do you think the next logical step here is to expose the client interface directly as dependency and potentially remove the need for this wrapper at all

Reply:
@maleck13 I could see that, from my perspective the helper methods make the calling code "cleaner", especially the NetNamespaces ones. Images and Rules I could see differently but I am not rushing to remove the functions.